samples/subsys/fs/ext2/README.rst

```diff
@@ -0,0 +1,61 @@
+Ext2 Sample application
+#######################
+
+Overview
+********
+
+This sample app uses the Ext2 file system driver to mount an SDHC card connected over an SPI bus.
+
+Requirements
+************
+
+This sample requires SD card support and a microSD card formatted with Ext2 file system.
+Restrictions:
+
+* max file system size is:
+
+   .. code-block:: none
+
+      - 8MB for 1k blocks
+      - 32MB for 2k blocks
+      - 128MB for 4k blocks
+
+* file system has to be created with specific options (the implementation doesn't support all ext2
+  features). See example commands below.
+
+
+Building and running
+********************
+
+Preparing SD card
+-----------------
+
+To be able to use SD card you have to create a partition with ext2 file system of proper size
+(according to size limitations described above). Example commands for preparing SD card visible as
+``/dev/sdc``, 4K blocks and 128MB size:
+
+1. Prepare partition with 128MB size:
```

Comment:
Would mention that su/sudo will be needed here and add a note that they should not blindly use /dev/sdc and must determine their sd card device properly, don't want someone running this then asking where their 1TB of projects has vanished to